openapi.json

```diff
@@ -1502,6 +1519,116 @@
             "example": "deploymentconfig"
           }
         }
+      },
+      "PlotsData":{
+        "type": "object",
+        "properties":{
+          "datapoints":{
+            "type": "integer",
+            "example": 4
+          },
+        "plots_data":{
+          "$ref": "#/components/schemas/PlotDetails"
+          }
+        }
+      },
+      "PlotDetails": {
+        "type": "object",
+        "properties":{
+          "2023-04-01T06:00:00Z":{
+            "type": "object",
+            "properties":{
+              "cpuUsage":{
+                "$ref": "#/components/schemas/cpuUsage"
+              },
+              "memoryUsage":{
+                "$ref":"#/components/schemas/memoryUsage"
+              }
+            }
+          },
+          "2023-04-01T12:00:00Z":{
+            "type": "object",
+            "properties":{
+              "cpuUsage":{
+                "$ref": "#/components/schemas/cpuUsage"
+              },
+              "memoryUsage":{
+                "$ref":"#/components/schemas/memoryUsage"
+              }
+            }
+          },
+          "2023-04-01T18:00:00Z":{
+            "type": "object",
+            "properties":{
+              "cpuUsage":{
+                "$ref": "#/components/schemas/cpuUsage"
+              },
+              "memoryUsage":{
+                "$ref":"#/components/schemas/memoryUsage"
+              }
+            }
+          },
+          "2023-04-02T00:00:00Z":{}
```

Comment:
This one is showing up as a string on swagger UI.

```
"2023-04-02T00:00:00Z": "string"
```
Could you re-check by including following for the same?

```
        "type": "object",
        "additionalProperties": false
```